packages/commerce-sdk-react/CHANGELOG.md

```diff
@@ -1,9 +1,4 @@
-## v2.0.0-preview.0 (Jun 21, 2024)
-## v2.0.0-dev (Jun 21, 2024)
-## v3.6.0-preview.0 (Jun 21, 2024)
-## v2.0.0-dev (Jun 21, 2024)
-## v3.6.0-dev.1 (Jun 21, 2024)
-## v2.0.0-dev (May 21, 2024)
+## v2.0.0 (Jun 25, 2024)
```

Comment:
Should this be v2.0.0.preview.1 to be consistent with package and lock files? 🤔 These will get published to npm, so I think we should keep them consistent.

Reply:
Nah, I don't think it's necessary :) Our aim for the preview versions is to release 2.0.0. 

Yes, they get published to npm, but I think they will be used by us internally only.